README.md

```diff
@@ -5,7 +5,7 @@ A simple chat app using Angular and socket.io
 
 [simple-chat Deployment](https://github.com/mbrandau/simple-chat/blob/master/frontend/README.md) 
```

Comment:
Just leave that section out. It'll be covered in the contribution guide.

Reply:
You mean remove the section right?

Reply:
Correct :+1: